how-to-respond-en.md

```diff
@@ -0,0 +1,43 @@
+
+Our goal is to create a community in which government, suppliers and partners work together. A community, without lock-ins, that works for all parties involved. It is also in this collective spirit that we’re setting up this market consultation.
+
+We want to use your experience, inspiration and brainpower to come to the right set-up. In order to be able to have the right conversation, we have set out our needs, proposals and questions to the market in this document.
+
+## Procedure of the market consultation
+
+The market consultation consists of two parts:
+
+1. A written response to our questions. We're deliberately not restricting the form and length of your answers. These responses should be received by 1-5-2020 before 17:00. You can send your answers to niels.lindeboom@dimpact.nl. Based on your responses we will review our proposal and adjust it if there are good suggestions. This modified proposal will then become the starting point for the workshop (part two of the consultation).
+2. Optional: one or more workshops where together we will determine the various needs and prerequisites within the community (participation only on request of OpenZaak). After submitting your answers, you will receive a message no later than 13-5-2020 if you are invited for the workshop. This will take place on 20th and 21st of May 2020. With the invitation you will receive further information about the exact time and location.
+
+Dimpact is responsible for the communication about this market consultation. You can contact Niels Lindeboom at <niels.lindeboom@dimpact.nl> or 06-22270923.
+
+## Market consultation report
+
+We will produce an anonymized report after interested parties have answered the questions from the market consultation document and the workshop has taken place. This report will be made available to all market consultation participants, and will be published with the announcement of a possible tender.
+
+## Planning
+
+| Description | Date |
+|-------------|-------|
+| Market consultation publication | 14-4-2020 |
+| Submission deadline | 1-5-2020, 17:00 |
+| Possible invitation to workshop | 13-05-2020 |
+| Workshop (invitation only) | 20/21-5-2020 |
+
+## Legal aspects
+
+This document is for market consultation purposes only. It is made available on the express condition that it will only be used to provide suggestions and ideas. Participation in this market consultation cannot lead to obligations on the part of the participating market parties to enter into agreements with the participating municipalities or Dimpact, and vice versa. Market consultation participation does not entail any reimbursement of costs.
+
+| <a href="README.md">Home</a> | <a href="introduction-en.md">Introduction</a> | <a href="introduction-en.md">Our proposals and questions</a> | How to respond |
```

Comment:
<a href="introduction-en.md">Our proposals and questions</a>

Link to be fixed